test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/ReadinessProbeIntegrationTest.java

```diff
@@ -113,4 +148,33 @@ public void testKieServerReadinessProbe() {
         }
         Assertions.fail("Timeout while waiting for OpenShift router to establish connection to Kie server.");
     }
+
+    private boolean isBCLoginScreenAvailable() {
```

Comment:
I would rather replace return value with validations. Return value hides line number where the issue occurs.